nrfx/drivers/include/nrfx_lpcomp.h

```diff
@@ -54,6 +54,25 @@ extern "C" {
  */
 typedef void (* nrfx_lpcomp_event_handler_t)(nrf_lpcomp_event_t event);
 
+/** @brief LPCOMP shortcut masks. */
+typedef enum
+{
+    NRFX_LPCOMP_SHORT_STOP_AFTER_CROSS_EVT = NRF_LPCOMP_SHORT_CROSS_STOP_MASK,   ///< Shortcut between the CROSS event and the STOP task.
+    NRFX_LPCOMP_SHORT_STOP_AFTER_UP_EVT    = NRF_LPCOMP_SHORT_UP_STOP_MASK,      ///< Shortcut between the UP event and the STOP task.
+    NRFX_LPCOMP_SHORT_STOP_AFTER_DOWN_EVT  = NRF_LPCOMP_SHORT_DOWN_STOP_MASK,    ///< Shortcut between the DOWN event and the STOP task.
+    NRFX_LPCOMP_SHORT_READY_STOP_MASK      = NRF_LPCOMP_SHORT_READY_STOP_MASK,   ///< Shortcut between READY event and STOP task.
+    NRFX_LPCOMP_SHORT_READY_SAMPLE_MASK    = NRF_LPCOMP_SHORT_READY_SAMPLE_MASK, ///< Shortcut between READY event and SAMPLE task.
```

Comment:
I don't think it's reasonable to introduce such inconsistent naming here (some values with suffix `_EVT` and some with `_MASK`). I assume this was done to follow the naming used in `nrfx_comp.h`, but in my opinion it's a wrong approach. The `nrfx_comp_short_mask_t` and `nrfx_comp_evt_en_mask_t` enumerations are unneeded (`nrf_comp_int_mask_t` and `nrf_comp_short_mask_t` can be used instead) and in fact inconsistent with the rest of nrfx - no other nrfx driver defines such enumerations, they use those from HAL instead.
My proposal is to not introduce `nrfx_lpcomp_short_mask_t` and `nrfx_lpcomp_evt_en_mask_t`, but instead to deprecate `nrfx_comp_short_mask_t` and `nrfx_comp_evt_en_mask_t`.